.travis/override.trusty.main

```diff
@@ -0,0 +1,2 @@
+appliance50	optional	cs50
+teamviewer9	optional	cs50
```

Comment:
> Hm, aren't you only pushing to http://mirror.cs50.net/ide50/2015/dists/trusty/main/binary-amd64/?
I am.

> I do see teamviewer in there (which can be deleted, shouldn't be there). But I don't see appliance50 there?

Hm, must've been deleted from mirror then. This file, along with the script, were originally on local cdn.